queue/abstract.lua

```diff
@@ -63,6 +63,26 @@ queue.driver = {
     limfifottl  = require('queue.abstract.driver.limfifottl')
 }
 
+local function tube_release_all_tasks(tube)
+    local prefix = ('queue: [tube "%s"] '):format(tube.name)
+
+    -- We lean on stable iterators in this function.
+    -- https://github.com/tarantool/tarantool/issues/1796
+    if not qc.check_version({1, 7, 5}) then
+        log.error(prefix .. 'no stable iterator support: skip task releasing')
```

Comment:
It's look like warning or info, not error

Reply:
queue is not operational in the case. It is the error in my understanding.